include/deal.II/base/symmetric_tensor.h

```diff
@@ -959,9 +963,10 @@ namespace internal
 
 
     template <int rank_, int dim, bool constness, int P, typename Number>
-    DEAL_II_CONSTEXPR inline Accessor<rank_, dim, constness, P - 1, Number>
-      Accessor<rank_, dim, constness, P, Number>::
-      operator[](const unsigned int i)
+    DEAL_II_CONSTEXPR inline DEAL_II_ALWAYS_INLINE
```

Comment:
Do we need `inline` and `DEAL_II_ALWAYS_INLINE`? We don't put both together elsewhere.

Reply:
Actually, it looks like most of the declarations after this one put both together (just not the ones before this)

Reply:
Since these are templates, declaring them as `inline` does probably not change anything.

Reply:
For templates `inline` is indeed not necessary. Should we remove them in the tensor files (it would certainly not hurt readability here)? However, on general header file implementations we cannot merge `DEAL_II_ALWAYS_INLINE` with `inline` because the former might expand to nothing.

Reply:
I did not realize that `DEAL_II_ALWAYS_INLINE` could be blank; this makes sense.